src/appleseed.studio/mainwindow/rendering/rendertab.cpp

```diff
@@ -498,7 +498,6 @@ void RenderTab::recreate_handlers()
     // Set initial state.
     m_pixel_inspector_handler->set_enabled(false);
     m_camera_controller->set_enabled(false);
-    m_scene_picking_handler->set_enabled(true);
```

Comment:
Have you tested different cases to ensure that the scene picker is enabled when it should ? 
- Start/stop rendering
- Interactive rendering
- Abort rendering
- Open a new project, start a new project

Reply:
Yes I've tested with all the cases.

If you look closely you would find that this line is redundant because just a couple of lines above when we create a `ScenePickingHandler`, the `m_enabled` member of it is set to true (in constructor).